src/components/ScreenStackHeaderConfig.tsx

```diff
@@ -109,5 +109,6 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center',
+    minHeight: '100%',
```

Comment:
Doesn't it set the header config height to the height of its parent? We don't want the header to be of full screen height, do we?

Reply:
I thought It's taking the height available in the AppBarLayout, but after opening it in the layout inspector it really is the height of the screen, just positioned strangely:
![Screenshot 2024-10-28 at 11 41 35](https://github.com/user-attachments/assets/432bdf2c-104e-4574-88c5-b359d36ea8eb)

Got CI failures because of it as well, I'm gonna change it 😉